proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -441,6 +441,7 @@ message GetChecksByFederatedGraphNameRequest {
   string startDate = 4;
   string endDate = 5;
   string namespace = 6;
+  repeated string subgraphs = 7;
```

Comment:
It is not clear to me that we have applied a filter. Could you come up with a better name? e..g `include_subgraphs`